examples/experimental/net-user-v2.yaml

```diff
@@ -0,0 +1,15 @@
+# Example to run vz instance with experimental user-v2 network enabled
```

Comment:
Can we test this in the CI?
Probably we can just use Linux instances on Cirrus (faster than GHA).

Reply:
Done, added as part of current it test script itself. 

Missed some condition changes. Will update this and make the pipeline green in linux.

Reply:
Thanks, I'd like to see VM-to-VM communication test, but that can be a separate PR

Reply:
This is done as well. For now its one way ping from one vm to another. But can improve this later.